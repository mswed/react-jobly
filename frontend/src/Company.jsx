import { useParams } from 'react-router-dom';
const Company = ({ name, description, jobs }) => {
  return (
    <div>
      <h1>Company Name</h1>
      <h2>Description</h2>
      <div>List of company jobs</div>
    </div>
  );
};

export default Company;
