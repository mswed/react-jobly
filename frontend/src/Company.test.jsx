import { it, expect } from 'vitest';
import { render } from '@testing-library/react';
import Company from './Company';

it('renders without crashing', async () => {
  render(<Company />);
});

it('matches snapshot', async () => {
  const { asFragment } = render(<Company />);

  expect(asFragment()).toMatchSnapshot();
});
