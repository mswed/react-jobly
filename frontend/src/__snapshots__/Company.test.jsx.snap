// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <h1>
      Company Name
    </h1>
    <h2>
      Description
    </h2>
    <div>
      List of company jobs
    </div>
  </div>
</DocumentFragment>
`;
